src/test/tests_common.h

```diff
@@ -252,4 +290,80 @@ tc_sem_timedwait(sem_t *sem, int sec, int line_number)
 	D_ASSERTF(rc == 0, "sem_timedwait() failed at line %d rc: %d\n",
 		  line_number, rc);
 }
+
+void
+psr_start_basic(crt_context_t *crt_ctx, pthread_t *progress_thread)
```

Comment:
i think we should rename this function. First, 'psr' has special meaning in cart, which isnt related to what we do here, and second, i think we shoudl prefix all our test common stuff with 'tc_'